src/fraction.rs

```diff
@@ -180,5 +180,12 @@ impl Default for Fraction {
     }
 }
 
+#[cfg(feature = "defmt")]
+impl defmt::Format for Fraction {
+    fn format(&self, fmt: defmt::Formatter) {
+        defmt::write!(fmt, "{} / {}", self.numerator(), self.denominator())
```

Comment:
So, usually Format is automatically generated in a similar format to `Debug`, which means if we would copy the pattern 1:1 this should be `Fraction(Ratio {{num: {}, delim {}}})`. But I think your solution is fine, not sticking to closely to the pattern.  Definitely better readable

Reply:
I chose to do it this way because `Ratio`'s members are not public, and if their implementation changes it would no longer be accurate (though that is unlikely).  I felt like this was the best solution for readability & accuracy of what it represents.